docs/getting-started.md

```diff
@@ -186,6 +186,54 @@ spec:
 EOF
 ```
 
+Also, after a `CloudProviderAccount` CR is added, VPCs are automatically polled
+for the configured region. From this VPC inventory, VPC ID or VPC Name can be
```

Comment:
we dont need this statement.. keep it short. From this VPC inventory, VPC ID or VPC Name can be... VPC of interest.